lib/rules/jsx-indent.js

```diff
@@ -165,8 +184,9 @@ module.exports = {
       } while (token.type === 'JSXText' && /^\s*$/.test(token.value));
       var startLine = node.loc.start.line;
       var endLine = token ? token.loc.end.line : -1;
+      var whitespaceOnly = token ? /\n\s*$/.test(token.value) : false;
```

Comment:
This probably doesn't matter, but I think this regex won't match the first line? Should it be `/^\s*$/` instead? I'm also not sure what `token.value` represents, so perhaps the `\n` is necessary. (see also the regex on line 184)

If these end up being the same, it might not be a bad idea to move them to a constant at the module level.